parm/config/gfs/config.base

```diff
@@ -184,6 +184,7 @@ export DO_AERO="NO"
 export WAVE_CDUMP="" # When to include wave suite: gdas, gfs, or both
 export DOBNDPNT_WAVE="NO"
 export FRAC_GRID=".true."
+export CCPP_SUITE="FV3_GFS_v17_p8_ugwpv1"
```

Comment:
Is there a reason to put the `CCPP_SUITE` in `config.base` rather than `config.ufs`? `config.base` is not a catch-all for all user-configurable settings; this should only be done if it is needed by additional jobs that are not using `config.ufs`.

Reply:
The only reason I have right now is that in many situations, one only need to modify one config file for new experiments.

I will remove this change. Thanks!